presto-main/src/main/java/io/prestosql/connector/system/jdbc/ColumnJdbcTable.java

```diff
@@ -94,6 +94,10 @@
     public static final SchemaTableName NAME = new SchemaTableName("jdbc", "columns");
 
     private static final int MAX_DOMAIN_SIZE = 100;
+    private static final int MAX_TIMEZONE_LENGTH = ZoneId.getAvailableZoneIds().stream()
```

Comment:
Looks reasonable to me. Is it what eg Oracle does? they support named zones as well.

The assumption is that SystemV zones do not have long names, right?

Reply:
I haven't tried with Oracle. Let me check that.

Reply:
Oracle seems broken. It returns `13` as the display size for timestamp with time zone regardless of the precision. (13 can't even fit a timestamp with precision = 0 -- maybe if you represent it as `+YYMMDDHHMMSS`). Also, `getPrecision` returns 0 always.